sdk/helper/keysutil/policy_test.go

```diff
@@ -28,6 +28,118 @@ import (
 	"golang.org/x/crypto/ed25519"
 )
 
+var allTestKeyTypes = []KeyType{
```

Comment:
Maybe put a warning here never to change the order of the elements?

Reply:
Done within https://github.com/hashicorp/vault/pull/27156/commits/1aab0588fbc54f3c6f31dc7f73c089987cea0c5b